plugins/Toolbox/resources/qml/dialogs/CompatibilityDialog.qml

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Toolbox is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Dialogs 1.1
+import QtQuick.Window 2.2
+import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
+
+// TODO: Switch to QtQuick.Controls 2.x and remove QtQuick.Controls.Styles
+
+import UM 1.1 as UM
+import Cura 1.6 as Cura
+
+
+UM.Dialog{
```

Comment:
This dialog needs some serious polishing to display the packages as shown in CURA-7038's chart. WIP.